doc/bp-Command.md

```diff
@@ -0,0 +1,231 @@
+## ``bp`` - The BiblioPixel Project Runner
+
+``bp`` is a command-line script installed with Bibliopixel.  It can run
+projects and demos, configure hardware, save and load defaults, and more.
+
+``bp`` takes one of 16 *commands*:
+  all_pixel, all_pixel_test, clear_cache, color, demo, devices, info, load
+  list, remove, reset, run, save, set, show, update
+
+Some commands take *arguments* - often names of files.
+
+### Examples of ``bp`` commands and arguments
+
+``````
+bp info
+bp demo circle
+bp run foo.json bar.json
+bp foo.json bar.json      # run is the default command
+``````
+
+The commands are summarized below, and you can get more detailed help about
+each one by typing:
+
+``````
```

Comment:
Really, that six ticks command is truly, truly horrible.  There has to be another way...!  What's wrong with these people?

Reply:
Again, this is a file that was not supposed to be here. It's not actually 6 ticks, because I fixed that logic in command.py